src/app/RatingCalculator.py

```diff
@@ -0,0 +1,24 @@
+__author__ = 'Alex'
+
+
+class RatingCalculator():
+    def __init__(self, participant_q, participant_v, winner=None):
```

Comment:
these are just the ratings of the participants. I was suggesting you pass in the entire participant entities, and then call put() on each at the end of process. i also think participants will have their own k-values in certain rating schemes anyway. i think this is fine for now except for not saving them.